src/sentry/static/sentry/app/views/routeNotFound.jsx

```diff
@@ -8,7 +8,9 @@ import NotFound from '../components/errors/notFound';
 
 class RouteNotFound extends React.Component {
   componentDidMount() {
-    Raven.captureException(new Error('Route not found'));
+    Raven.captureException(new Error('Route not found'), {
+      fingerprint: ['RouteNotFound'],
```

Comment:
i guess theres no other mechanism for fingerprints here so this is probably good